### ISSUE TITLE
Spring Cloud Commons Bootstrap is not enabled by default for Spring-Cloud-2022.0

### ISSUE THREAD

**Issue by fdurantsq** [spring-cloud/spring-cloud-release#273]
**Describe the bug**
Really I'm just suggesting changes to breaking changes list, and to add `spring-cloud-bootstrap-starter` to spring-cloud-commons documentation so people can more easily make upgrades or get started with using Bootstrap configurations. Specifically:

> spring.config.use-legacy-processing=true no longer enables bootstrap because of changes in Spring Boot 3.0. Please use spring.cloud.bootstrap.enabled=true.

Does not adequately describe the side-effects of the changeset [here](https://github.com/spring-cloud/spring-cloud-commons/commit/a183bc709f7e332ceb0d4d1cf8d603d93917581b#diff-3f78fa4206effdec3eb462fb43cafc8a87dded591234cd2b075b9ea6fca73b79R100)

Source: https://github.com/spring-cloud/spring-cloud-release/wiki/Spring-Cloud-2022.0-Release-Notes

**Steps to reproduce:**
- Use a spring cloud project prior to 2022, using a spring.factories class pointing to BootstrapConfiguration files. Do not use spring-cloud-starter-bootstrap
- Upgrade to 2022, following the migration guide

**Expectations**:
- Breaking changes list above describes that "spring.cloud.bootstrap.enabled does not default to true anymore." Users must.... [see Actual Behavior]"
- Add section that "spring-cloud-starter-bootstrap" will auto enabled bootstrap?

**Actual behavior:**
- BootstrapConfiguration does not load, until a system property is set as described above, or spring-cloud-bootstrap-starter is added which adds the Marker.java class that enabled BootstrapConfiguration initialization again.

**Comment by spencergibb** [comment]
I disagree. Both methods should work. If the property does not work it is a bug. How are you setting the property?

**Comment by spencergibb** [comment]
I didn't fully read the last comment. It has to be a system property, environment variable or cli argument
